From f4578eaf711f3d5e15f3e5fdb98cce3ebae80a2b Mon Sep 17 00:00:00 2001
From: Gal Pressman <galp@mellanox.com>
Date: Thu, 28 Sep 2017 15:08:52 +0300
Subject: [PATCH 10/11] ethtool.8: Fix formatting of advertise bitmask

Fields should be separated with a tab instead of spaces.
Also, remove an accidental ' character from 2500baseX Full mode.

Fixes: 64dfc5e2f046 ("ethtool: Add support for 2500baseT/5000baseT link modes")
Signed-off-by: Gal Pressman <galp@mellanox.com>
Reviewed-by: Tariq Toukan <tariqt@mellanox.com>
Signed-off-by: John W. Linville <linville@tuxdriver.com>
(cherry picked from commit f35a4bcd55164495deb4eef0975a1623d760ff91)
---
 ethtool.8.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ethtool.8.in b/ethtool.8.in
index eb0f551..c7208ab 100644
--- a/ethtool.8.in
+++ b/ethtool.8.in
@@ -578,9 +578,9 @@ lB	l	lB.
 0x020	1000baseT Full
 0x20000	1000baseKX Full
 0x20000000000	1000baseX Full
-0x800000000000  2500baseT Full
-0x8000	2500baseX Full	(not supported by IEEE standards)'
-0x1000000000000  5000baseT Full
+0x800000000000	2500baseT Full
+0x8000	2500baseX Full	(not supported by IEEE standards)
+0x1000000000000	5000baseT Full
 0x1000	10000baseT Full
 0x40000	10000baseKX4 Full
 0x80000	10000baseKR Full
-- 
1.8.3.1

